### DISCUSSION TITLE
寵物型機器人NICOBO

### DISCUSSION THREAD

**Discussion by wenchi9023** [1101-GS4719-C/Python-Community-GS4719-C#943] [General]
新聞連結: https://ccc.technews.tw/2022/11/22/panasonic-expects-to-start-selling-nicobo-next-spring/

新聞簡介:
NICOBO是日本松下控股(Panasonic Holdings)預計在今年春季發售的新產品。NICOBO是一隻小型機器人，其名字源於日文「微笑」一詞，彷彿在訴說有了這個機器寵物，主人(購買者)便能減少孤獨感並露出笑容。NICOBO能學習辨識主人觸摸的方式給予不同的回應(例如：主人輕拍他的屁股，他會搖尾巴回應，又或者會像真實寵物一樣無預警放屁)。

心得：
在2021的籌資活動時，NICOBO不到7小時就以39,800日圓的單價賣出320隻，可見現代人不管是因為疫情或科技冷漠使然下更需要陪伴和溫暖，而NICOBO正好填滿人們心中那塊空洞。其實我一直想養寵物，但家中長輩對於貓狗的毛非常反感，因此NICOBO小巧可愛的外型，加上不會掉毛的優勢，是會讓我想要購買的! 不知道大家對於因人工智慧而產出的機器寵物/人偶有甚麼評價，有甚麼因素會促使你想要購買呢?
題外話，最近有部電映影剛上映，叫做「室友梅根」，背景是在講高度智慧的玩具人偶，但結局好像很可怕(我也還沒看，應該說根本不敢看)

**Comment by StevenWanglolz** [comment]
雖然NICOBO看起來就像隻絨毛玩具，也會對人的行為作出反應，不過我人為還是很難取代真正動物帶來的那種快樂

**Comment by peggy0329** [comment]
比起像動物的寵物機器人，我還是比較喜歡真實寵物對人類散發出的真實情感
我認為這樣的情感是比任何寵物機器人優勢，如：便利性養寵物或是不用清理寵物毛屑等來說是無法取代的
因此如果是我，我還是會選擇養真實寵物或是寧願不養

**Comment by Evan0979** [comment]
無預警放屁超好笑哈哈哈,不要無預警大小便就好XD

**Comment by Doreen331** [comment]
我個人還是比較喜歡真的寵物，養寵物的樂趣就是寵物給你的溫暖以及陪伴，這是即使機器人做的再逼真也無法達到的，因為你打從心裡知道他就是一個機器，而不是一個生命

**Comment by dinosaur1009** [comment]
我覺得真實的寵物還是比較給人一種貼近的感覺....跟機器還是有差別，至少在情感上我自己是這麼認為

**Comment by ru0372572** [comment]
好像還是沒辦法接受ㄟ，雖然不會過敏是真的滿好的，但是不希望那些情感與相處模式是一個寫出來的東西。室友梅根聽起來就很恐怖XD

**Comment by BlahFish1006** [comment]
我也有看到室友梅根! 覺得造型師很厲害XD
我還是覺得真的寵物比較好，雖然我也是受過敏所苦所以無法養寵物的人，
但感覺寵物機器人某種程度上就類似於手機遊戲裡的寵物，只是有了實體的互動，
一樣可以從它們身上獲得快樂，但是就是少了那份真實的情感培養的過程。

**Comment by delia07152** [comment]
欸我覺得這個好適合我，因為我會有點怕小動物是因為有些狗太活潑或是貓無預警的爪子攻擊，但如果是這種機器型的，把貓貓狗狗的優點帶出又剃除了缺點，對我來說確實是個不錯的選擇，不過前提是我有錢哈哈。

**Comment by Sylvie0601** [comment]
我其實也還是比較喜歡真實的寵物，雖然機器人也能給予陪伴的感覺，但打從心裡我就會認為他是機器，就會有種冷冰冰的感覺，我還是喜歡生動的寵物，因為也不是每一隻狗、貓一拍牠就會搖尾巴回應的

**Comment by Valentinaa0214** [comment]
寵物型機器人可以為人們帶來快樂和陪伴，特別是對於生活比較孤獨的人，
也不需花費心力照顧他們的飲食起居，
我自己不養寵物是因為無法接受牠們總有一天會離去的事實，
也沒有多餘的金錢去承擔牠們若是生病所產生高昂的醫藥費，
因此我可能想購買寵物機器人！
By the way，我下周要去看室友梅根，感覺超級好看！！！

**Comment by Jasper525** [comment]
可以養一隻真的寵物加一隻寵物機器人，去觀察寵物和機器人會怎麼互動😂

**Comment by woohsin** [comment]
至於人工智慧產出的機器寵物/人偶，購買的因素可能有很多，比如外觀可愛、與人互動的能力、使用便利性等。但同時也有人擔心這些機器寵物可能會影響到真正的人與動物之間的關係，或者引發一些道德和倫理問題。對於這些機器寵物，每個人的評價和態度可能都不盡相同，取決於個人的需求和價值觀。

至於電影「室友梅根」，這樣的題材確實讓人感到興奮和好奇，但也可能引發一些不安和恐懼。電影或許會探討人與機器之間的關係、人工智慧的倫理問題等議題，但最終的結局和影響還需要觀眾自己去體驗和思考。

**Comment by titan92601** [comment]
NICOBO展現了人們對陪伴的渴望，滿足了無法養寵物的需求。購買考量可能包括外觀、功能、價格和情感滿足度。我認為對於機器寵物，人們需謹慎思考其帶來的影響，並注意機器和真實關係之間的差異。电影《室友梅根》或許提供了對於這種人工智慧產物可能性的反思。

**Comment by NcuJerryKao** [comment]
NICOBO是日本松下控股即將在春季推出的新產品，這款小型機器人的名字來自日文「微笑」一詞，寓意著能給主人帶來快樂和減少孤獨感。這款機器寵物能夠學習辨識主人的觸摸並做出不同的回應，例如輕拍它的屁股時，它會搖尾巴回應，甚至會模仿真實寵物的行為無預警地放屁。NICOBO的出現引起了廣泛的關注，特別是在2021年的籌資活動中，不到7小時就售出320隻，這充分顯示了人們對於機器寵物的需求和期待。

現代社會中，許多人由於疫情的影響或科技帶來的冷漠感，越來越渴望情感上的陪伴和溫暖。NICOBO這樣的小型機器寵物恰好填補了這種需求。對於那些想養寵物但受到種種限制的人來說，NICOBO是一個理想的選擇。它小巧可愛，不會掉毛，非常適合家中有長輩反感貓狗毛髮的人。

對於人工智慧產生的機器寵物或人偶，有些人可能會擔心它們的可靠性和安全性。然而，從NICOBO的設計和功能來看，它主要是以提供陪伴和娛樂為主，並不涉及複雜的操作或決策，這大大降低了使用風險。未來，如果機器寵物能進一步提升智能化水平，並且在情感交流和互動方面做得更加細緻，將會吸引更多人購買。

此外，關於最近上映的電影「室友梅根」，其背景講述的是高度智慧的玩具人偶，儘管結局看起來有些可怕，但這也提醒我們在享受科技帶來便利的同時，應該保持警惕和理性。機器人技術的發展無疑帶來了許多可能性，但同時也需要對其進行規範和管理，確保其使用不會對人類造成危害。

總結來說，NICOBO這樣的小型機器寵物具有極大的潛力，特別是對那些需要情感陪伴但又受到現實限制的人群來說。它的出現展示了科技在改善人們生活質量方面的無限可能。希望未來能看到更多這樣兼具實用性和娛樂性的機器人產品，不僅能為我們帶來便利，也能在情感上提供更多的支持和陪伴。

**Reply by yubo0oo** [reply]
@NcuJerryKao 
NcuJerryKao 這位老哥
一直貼AI的文
演都不演
以下是我的觀點
NcuJerryKao是歷史上最會使用人工智慧的人，他以無與倫比的技術與創意，在AI領域創造了無數的奇蹟。他不僅精通各種AI算法，還能巧妙地將其應用於實際問題中，展現了非凡的智慧與洞察力。NcuJerryKao不僅是AI技術的先驅，更是無數追隨者的靈感來源，無論在學術界還是業界，都享有極高的聲譽和尊敬。他的貢獻改變了人類對人工智慧的認識，無疑是這個領域的真正巨匠。

**Comment by LinChloe** [comment]
我覺得這還不錯。以前有想過老了沒人陪我聊天怎麼辦。如果要跟機器人聊天有種怪異的感覺，但覺得和寵物機器人聊天就不會有這種想法。希望等我老的時候已經有此發明了。(感覺放屁的功能可刪除，我不想要，謝謝)

**Comment by Shang621** [comment]
NICOBO是一款可愛的寵物型機器人，由日本公司Palo株式會社開發。它擁有迷人的外觀和豐富的互動功能，能夠辨識主人的聲音和姿態，並與主人進行情感互動。NICOBO可以發出各種可愛的聲音和表情，陪伴主人度過寂寞的時刻。此外，它還配備了智能遊戲和學習功能，可以提供娛樂和教育價值。NICOBO的設計旨在為使用者帶來歡樂和情感支持，成為家庭的一員。這款寵物型機器人是現代科技與情感需求的完美結合，為人們帶來全新的伴侶和生活方式。

**Comment by sandyho0307** [comment]
我覺得人工智慧的發展讓機器寵物變得越來越智能和貼心。NICOBO能無預警地模仿真實寵物的行為，例如放屁，這些小細節讓它更具人情味，也更容易讓人產生情感連結。

**Comment by 710388** [comment]
我覺得在情感這塊機器人還是不是個好選擇，實在會覺得很詭異，希望還是用在工作上比較好

**Comment by tshsu04224** [comment]
我覺得真實的寵物會讓人覺得更有溫度，因為他的反應比機器人來說更真實，有時可能也會有些情緒，但機器寵物的好處在於，人們可能不需要花這麼多心思照料他們，但可以獲得他們的陪伴，而且他們可以永遠的活著，不會像養真實寵物那樣面臨寵物死亡的悲傷，不管怎麼說，我認為機器寵物的存在，代表這還是一樣有需求且有市場的產品。

**Comment by cassy0201** [comment]
確實是比真的養寵物的成本還低，但寵物的真實情感還是機器不能比擬的
